js/ccf-app/src/crypto.ts

```diff
@@ -32,6 +32,11 @@ export const generateRsaKeyPair = ccf.generateRsaKeyPair;
  */
 export const wrapKey = ccf.wrapKey;
 
+/**
+ * @inheritDoc CCF.verifySignature
+ */
+export const verifySignature = ccf.verifySignature;
```

Comment:
#2315, can this go under crypto now please, the ccf namespace is crowded enough.

Reply:
Done.